test/runtests.jl

```diff
@@ -2,12 +2,20 @@
 using FFTW
 using Base.Test
 
-# Make super sure we don't pull in Base names
 importall FFTW
-import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
+if VERSION >= v"0.7.0-DEV.602"
+    import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
```

Comment:
Part of the problem is that because of the way we do the `Base.DFT` deprecation, we never actually call `Base.deprecate(Main, :fft)` etcetera.  This is easy to fix.